src/java/org/apache/cassandra/db/memtable/AbstractMemtable.java

```diff
@@ -110,6 +112,16 @@ EncodingStats encodingStats()
         return statsCollector.get();
     }
 
+    public LifecycleTransaction getFlushTransaction()
```

Comment:
nit: These two methods need `@Override`

Reply:
If it's Ok with you I've added `@Override` to all the other overriden methods in this class. I can undo this if you think it's unnecessary.